clarifai/cli/deployment.py

```diff
@@ -0,0 +1,49 @@
+import click
+from clarifai.cli.base import cli
+from clarifai.client.nodepool import Nodepool
+from clarifai.utils.cli import display_co_resources
+
+
+@cli.group()
+@click.option(
+    '--nodepool_id', required=True, help='Nodepool ID for the Nodepool to interact with.')
+@click.pass_context
+def deployment(ctx, nodepool_id):
```

Comment:
Can we have `dep` as alias for here, or any other relevant alias. WDYT?

Reply:
Done as `dpl`